frontend/src/scenes/insights/EmptyStates/EmptyStates.tsx

```diff
@@ -135,44 +129,21 @@ export function InsightErrorState({ excludeDetail, title, queryId }: InsightErro
                 <h2>{title || 'There was an error completing this query'}</h2>
                 {!excludeDetail && (
                     <div className="mt-4">
-                        We apologize for this unexpected situation. There are a few things you can do:
+                        We apologize for this unexpected situation. There are a couple of things you can do:
                         <ol>
                             <li>
                                 First and foremost you can <b>try again</b>. We recommended you wait a few moments
                                 before doing so.
                             </li>
                             <li>
-                                <a
-                                    data-attr="insight-error-raise-issue"
-                                    href="https://github.com/PostHog/posthog/issues/new?labels=bug&template=bug_report.md"
-                                    target="_blank"
-                                    rel="noreferrer noopener"
-                                >
-                                    Raise an issue
-                                </a>{' '}
-                                in our GitHub repository.
-                            </li>
-                            <li>
-                                Get in touch with us{' '}
-                                <a
-                                    data-attr="insight-error-slack"
-                                    href="https://posthog.com/slack"
-                                    rel="noopener noreferrer"
-                                    target="_blank"
-                                >
-                                    on Slack
-                                </a>
-                                .
-                            </li>
-                            <li>
-                                Email us at{' '}
-                                <a
-                                    data-attr="insight-error-email"
-                                    href="mailto:hey@posthog.com?subject=Insight%20graph%20error"
+                                <Link
+                                    data-attr="insight-error-bug-report"
+                                    onClick={() => {
+                                        openSupportForm('bug', 'analytics')
+                                    }}
                                 >
-                                    hey@posthog.com
-                                </a>
-                                .
+                                    Submit a bug report.
+                                </Link>{' '}
```

Comment:
Just because I want rebase to happen and then the period won't be blue.
```suggestion
                                    Submit a bug report
                                </Link>.
```

Reply:
I think this commit broke code formatting 😅

Reply:
yes lol...